jekyll/_cci2/linux-vm-support-policy.adoc

```diff
@@ -0,0 +1,97 @@
+---
+contentTags:
+  platform:
+  - Cloud
+  - Server v4.x
+  - Server v3.x
+---
+= CircleCI Linux VM images support policy
+:page-layout: classic-docs
+:page-liquid:
+:page-description: CircleCI Linux VM images release, update, and deprecation policy
+:icons: font
+:experimental:
+
+[#overview]
+== Overview
+
+This document outlines the CircleCI Linux VM (Machine) Image release, update and deprecation policy. This policy applies to all CircleCI Linux VM (Machine) Images built for the Machine executor. Note that there is a separate [CUDA image support policy](https://circleci.com/docs/linux-cuda-images-support-policy/).
```

Comment:
This does not align with what we got approved

Reply:
i think it is important to include references and backlinks to a very similar image as this eliminates confusion for all parties and is part of the PR template checklist. I can bring this up with support or maybe Sebastian can chime in here?

Reply:
I am ok if @sebastian-lerner is ok for this one

Reply:
sorry, what's the confusion? what did get approved and what's the delta?

Reply:
@sebastian-lerner: the only delta from what was approved is the the addition of this line to differentiate this from the Linux CUDA image

`Note that there is a separate [CUDA image support policy](https://circleci.com/docs/linux-cuda-images-support-policy/).`

Reply:
okay seems fine to me